cookbooks/bcpc-hadoop/recipes/oozie_pack.rb

```diff
@@ -0,0 +1,98 @@
+#
+# Recipe to include hbase jar files into Oozie
+#
+%w{
+hbase
```

Comment:
Can we use Maven or something lighter weight, instead of installing the HBase package?